app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'user_id'
```

Comment:
- [ ] Foreign_key should be `author_id` not `user_id` please address that

Reply:
I think this comment should be optional. user_id is foreign key from the users table.

Reply:
you are right make sure you make the right reference